jenkins/deployment/community/community-build-cortx.groovy

```diff
@@ -46,7 +48,7 @@ pipeline {
                 export ACCESS_KEY=${ACCESS_KEY}
                 export KEY_NAME=${KEY_NAME}
                             rm -rvf /usr/local/bin/aws /usr/local/bin/aws_completer /usr/local/aws-cli >/dev/null 2>&1
-                            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && yum install unzip && unzip awscliv2.zip
+                            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && yum install unzip -y && unzip awscliv2.zip
```

Comment:
unzip package installation is already added in https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/tool_setup.sh#L23

Reply:
We have to use AWS CLI before running the tool_setup script to setup the access and secret key on the VM. Hence unzipping the file is required before running the too_setup script.